boa3_test/test_sc/built_in_methods_test/CountTupleDifferentNonPrimitiveTypes.py

```diff
@@ -0,0 +1,10 @@
+from typing import Tuple, Any
+
+from boa3.builtin import public
+
+
+@public
+def main() -> Tuple[int, int, int]:
+    a: Tuple[Any] = ([b'unit', 'test'], [b'unit', b'unit'], [123, 123], [True, False], [True, False])
+    b: Tuple[int, int, int] = a.count([b'unit', 'test']), a.count([123, 123]), a.count([True, False])
```

Comment:
The unit tests are failing with Python 3.7 because of this
Apparently the Python 3.7 grammar doesn't accept tuples without parenthesis.
```suggestion
    b: Tuple[int, int, int] = (a.count([b'unit', 'test']), a.count([123, 123]), a.count([True, False]))
```